README.md

```diff
@@ -171,7 +176,7 @@ public ones will show up just fine.
 > http://localhost:5678/
 
 ##Project
-Create .bowerrc file with content:
+Within your project, you can create a .bowerrc file containing:
```

Comment:
I think creating the .bowerrc file is mandatory, so the word ```can``` can be misleading.